judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/clarification/ContestClarificationRoleChecker.java

```diff
@@ -63,7 +63,7 @@ public boolean canViewOwnClarifications(String userJid, Contest contest) {
                 && contestTimer.hasStarted(contest, userJid);
     }
 
-    private boolean isSupervisorWithClarificationPermissionOrAbove(String userJid, Contest contest) {
+    public boolean isSupervisorWithClarificationPermissionOrAbove(String userJid, Contest contest) {
```

Comment:
So, instead of making this public, create a method with the intention instead, e.g. `canViewAllClarifications()`, and add tests as well.